react-app-drawer-component/my-app/src/DrawerButton.tsx

```diff
@@ -0,0 +1,9 @@
+import { FiMenu } from 'react-icons/fi';
+
+type MenuButtonProps = {
+  onDrawerOpen: Function;
+};
+
+export default function MenuButton({ onDrawerOpen }: MenuButtonProps) {
```

Comment:
You usually want the component name to match the file name. I'm not sure which one you should change, though!